demo/docker-agent/Dockerfile.acapy

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/hyperledger/aries-cloudagent-python:py3.9-0.12.2
+FROM ghcr.io/hyperledger/aries-cloudagent-python:py3.9-0.12.3
```

Comment:
@jamshale -- should I be updating these as part of a release PR?  Easy to do if that should be done...

Reply:
These are 0.12.x — shouldn’t these references be the 1.x?

Reply:
Yes. These should get upgraded. I'm not exactly sure what this image is for. I think it's just the base image for the demo which still gets all the local src code copied into it. Really it's just being used as a python 3.12 image so it's not a big deal.

The best would be for a follow up PR to update all these images in the repo after the image has been published. We can't do it as the release process because they aren't available yet.

Reply:
I guess it depends if the Dockerfiles are used in tests or not.  If not used in tests, we could do them with the updates.

Maybe we change them to use the python and not ACA-Py, if everything is being copied in?